third_party/accessibility/ax/platform/ax_platform_node_win.cc

```diff
@@ -5205,6 +5218,10 @@ std::optional<DWORD> AXPlatformNodeWin::MojoEventToMSAAEvent(
       return EVENT_OBJECT_SHOW;
     case ax::mojom::Event::kValueChanged:
       return EVENT_OBJECT_VALUECHANGE;
+#ifdef EVENT_OBJECT_TEXTSELECTIONCHANGED
```

Comment:
I'm assuming this was added in Win 8 or 10 maybe? Consider adding a doc comment pointing out why it's `ifdef`ed.

Reply:
Actually on second look we probably do not need it. This macro should be defined from Windows Server 2008 onwards.